services/quiz/src/pages/StudyDetailPage/components/StudyInfo/index.tsx

```diff
@@ -1,19 +1,17 @@
-import { Tag, Text, UserAvatar } from '@depromeet/toks-components';
+import { SSRSuspense, Tag, Text, UserAvatar } from '@depromeet/toks-components';
+import { ErrorBoundary } from '@toss/error-boundary';
+import { useSuspendedQuery } from '@toss/react-query';
 
-import { User } from '../../../../../utils/userUtils';
-import { Body, Footer, Header, Info, StudyTags } from './style';
-
-interface StudyInfoProps {
-  studyId: string;
-  title: string;
-  description: string;
-  studyTags: string[];
-  members: User[];
-}
+import { QUERY_KEYS } from 'constants/queryKeys';
+import { getStudyInfo } from 'pages/StudyDetailPage/remote/quiz';
 
-const makeStudyTags = (tagInfos: string[]) => [...tagInfos].map(tagInfo => <Tag value={tagInfo} />);
+import { Body, Footer, Header, Info, StudyTags } from './style';
 
-export function StudyInfo({ studyId, title, description, studyTags, members }: StudyInfoProps) {
+function StudyInfo() {
+  const makeStudyTags = (tagInfos: string[]) =>
+    [...tagInfos].map((tagInfo, index) => <Tag key={index} value={tagInfo} />);
```

Comment:
```suggestion
    tagInfos.map((tagInfo, index) => <Tag key={index} value={tagInfo} />);
```

도 같지 않나용? 추가적으로 `Tag.Row`를 활용하면 될 것 같습니다

Reply:
Tag.Row는 왜 안썼을까요 ㅋㅋ ㅠㅠ 요것도 수정해놓을게요!!

Reply:
https://github.com/depromeet/toks-web/pull/112/commits/70dc8c172cac02e672f0285147712a2a0a742d93
요 부분도 수정했습니다 -~